scripts/Makefile.qemu

```diff
@@ -26,4 +26,8 @@ run: zephyr
 	$(if $(QEMU_PIPE),,@echo "To exit from QEMU enter: 'CTRL+a, x'")
 	@echo '[QEMU] CPU: $(QEMU_CPU_TYPE_$(ARCH))'
 	$(if $(CONFIG_X86_IAMCU),$(ZEPHYR_BASE)/scripts/qemu-machine-hack.py $(KERNEL_ELF_NAME))
-	$(Q)$(QEMU) $(QEMU_FLAGS) $(QEMU_EXTRA_FLAGS) -kernel $(KERNEL_ELF_NAME)
+	$(Q)if [ "$(CONFIG_NET_TEST)" = "y" ]; then \
+	$(QEMU) $(QEMU_FLAGS) -kernel $(KERNEL_ELF_NAME); \
+	else \
+	$(QEMU) $(QEMU_FLAGS) $(QEMU_EXTRA_FLAGS) -kernel $(KERNEL_ELF_NAME); \
```

Comment:
who sets QEMU_EXTRA_FLAGS? Why dont you unset it in there instead of changing this here in a top level Makefile that has nothing to do with networking?
Maybe in samples/net/common/Makefile.ipstack?